analysis/reanalyze/examples/deadcode/src/Hooks.res

```diff
@@ -66,27 +68,6 @@ let componentWithRenamedArgs = (~_to, ~_Type, ~cb: cb) => {
   React.string(_to.name ++ _Type.name)
 }
 
-@genType @react.component
-let makeWithRef = (~vehicle) => {
-  let _ = 34
-  ref =>
-    switch ref->Js.Nullable.toOption {
-    | Some(ref) => <button ref={ReactDOM.Ref.domRef(ref)}> {React.string(vehicle.name)} </button>
-    | None => React.null
-    }
-}
-
-@genType
-let testForwardRef = React.forwardRef(makeWithRef)
```

Comment:
I wasn't able to port any of the `React.forwardRef` examples to v11, but since it's always been discouraged and with React v19 allowing refs as props anyway I guess it's fine